modules/apps/blogs/blogs-service/src/main/java/com/liferay/blogs/internal/search/spi/model/query/contributor/BlogsEntryKeywordQueryContributor.java

```diff
@@ -48,9 +50,19 @@ public void contribute(
 			booleanQuery, searchContext, Field.SUBTITLE, false);
 		_queryHelper.addSearchLocalizedTerm(
 			booleanQuery, searchContext, Field.TITLE, false);
+
+		QueryConfig queryConfig = searchContext.getQueryConfig();
+
+		queryConfig.addHighlightFieldNames(
+			_searchLocalizationHelper.getLocalizedFieldNames(
+				new String[] {Field.CONTENT, Field.SUBTITLE, Field.TITLE},
```

Comment:
if SUBTITLE is not being added to the query with addSearchLocalizedTerm, do we still want to add the localized version to the highlight fields?

Reply:
also, i think we have to consider if SUBTITLE is even part of the rendered search results (and it may be, but i dont know off the top of my head, and BlogsEntryModelSummaryContributor doesnt seem to have it). if its not, then i dont think we would need to add it as a highlighted field